common/source_model/src/main/scala/weco/catalogue/source_model/sierra/rules/SierraRulesForRequesting.scala

```diff
@@ -189,6 +191,21 @@ object SierraRulesForRequesting {
           "This item cannot be requested online. Please place a manual request."
         )
 
+      // These cases cover the lines:
+      //
+      //    v|i||79||=|harcl||
+      //    # line above added so that Offsite deepstore material is not requestable.  LS 13/06/24
+      //
+      case i
+          if i
+            .fixedField("79")
+            .containsAnyOf(
+              "harcl"
+            ) =>
+        NotRequestable.NeedsManualRequest(
```

Comment:
Why not `NotRequestable.ItemUnavailable` as used on l.65?

Reply:
Oops, should be that.